packages/website/ts/pages/governance/connect_form.tsx

```diff
@@ -220,20 +220,14 @@ export class ConnectForm extends React.Component<Props, State> {
     }
     public async getZrxBalanceAsync(owner: string): Promise<BigNumber> {
         utils.assert(!_.isUndefined(this._contractWrappers), 'ContractWrappers must be instantiated.');
-        const injectedProvider = await this._getInjectedProviderIfExistsAsync();
-
-        if (!_.isUndefined(injectedProvider)) {
-            const contractAddresses = getContractAddressesForNetworkOrThrow(this.networkId);
-            const tokenAddress: string = contractAddresses.zrxToken;
-            try {
-                const amount = await this._contractWrappers.erc20Token.getBalanceAsync(tokenAddress, owner);
-                return amount;
-            } catch (error) {
-                return ZERO;
-            }
+        const contractAddresses = getContractAddressesForNetworkOrThrow(this.networkId);
+        const tokenAddress: string = contractAddresses.zrxToken;
+        try {
+            const amount = await this._contractWrappers.erc20Token.getBalanceAsync(tokenAddress, owner);
```

Comment:
should we explicitly check for undefined here in an if statement above? it was not clear to me that this try/catch would also be catching the case where we are trying to grab `erc20Token` from an `undefined` `this._contractWrappers`

Reply:
As discussed we have an assert above this. This is to catch any network errors or anything else that could go wrong with an initialised contractWrappers.